plugins/.clang-format

```diff
@@ -0,0 +1,5 @@
+# This file exists for backward compatibility only.
+# Please use the root .clang-format file instead.
```

Comment:
I'd prefer just removing this file if not used

Reply:
agree these were whoopsies, fixed